pkg/constants/constants.go

```diff
@@ -208,3 +208,11 @@ const (
 	// byte in a TCP connection.
 	ProtocolTCPServerFirst = "tcp-server-first"
 )
+
+// ClusterDomain is a type for a multicluster
+type ClusterDomain string
```

Comment:
Implement a `String()` method receiver for this type so we can use `LocalDomain.String()` vs `string(LocalDomain)` for consistency.

Reply:
done!